openedx/features/caliper_tracking/transformers/team_transformers.py

```diff
@@ -66,7 +67,9 @@ def edx_team_learner_added(current_event, caliper_event):
         ))
     )
 
-    object_link = utils.get_team_url_from_team_id(current_event['referer'], current_event['event']['team_id'])
+    object_link = utils.get_team_url_from_team_id(current_event['referer'],
+                                                  current_event['event'][
+                                                      'team_id'])
```

Comment:
Please refactor this into more appropriate form.

Reply:
done.